README.md

```diff
@@ -5,18 +5,23 @@
 > This is my showcase of my recently works
 
 ![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot1.png)
+![screenshot](./app_screenshot2.png)
+![screenshot](./app_screenshot3.png)
+![screenshot](./app_screenshot4.png)
+![screenshot](./app_screenshot5.png)
 
 Setup and mobile version of my portfolio.
 
 ## Built With
 
-- HtML file
+- HTML file
 - CSS file
-- linter
+- Linter
 
 ## Live Demo
 
-[Live Demo Link](https://oliverscz.github.io/mobile-version-project/)
+[Live Demo Link](https://oliverscz.github.io/mobile-version-project/) 
```

Comment:
[OPTIONAL] The links of your navbar are currently linking to pages that don't exist and generate a 404 error. Either use a placeholder (#) until you develop this feature. Or link them to the proper sections of your page. You can do this by assigning id's to section tags and linking the href to the proper id (e.g. #about, #works, etc.)